composer.json

```diff
@@ -13,12 +13,14 @@
         "php": ">=5.4",
         "swarrot/swarrot": "~2.0",
         "psr/log": "~1.0",
-        "symfony/framework-bundle": "~2.4",
-        "symfony/console": "~2.4"
+        "symfony/framework-bundle": "~2.4|3.*",
+        "symfony/console": "~2.4|3.*"
     },
     "require-dev": {
         "phpunit/phpunit": "~4.5"
     },
+    "minimum-stability": "dev",
+    "prefer-stable": true,
```

Comment:
it would be better to change the minimum stability only in builds needing it (see how Assetic does it). Otherwise, there is no build ensuring that dependencies can be resolved using only stable releases (which is what users of the bundle need by default)